src/manifestParser/manifestV2.ts

```diff
@@ -99,6 +116,20 @@ export default class ManifestV2 extends ManifestParser<Manifest> {
       });
     });
 
+    (result.manifest.web_accessible_resources ?? []).forEach((resource) => {
+      if (this.pluginExtras.webAccessibleScriptsFilter(resource)) {
+        const parsedWebAccessibleScript = this.parseOutputContentScript(
+          resource,
+          result,
+          bundle
+        );
+
+        webAccessibleResources.add(parsedWebAccessibleScript.scriptFileName);
```

Comment:
Very likely would also want to add `parsedWebAccessibleScript.webAccessibleFiles` to `webAccessibleResources` after this